app/src/main/java/com/example/myapplication/csv_to_json.py

```diff
@@ -0,0 +1,35 @@
+import csv, json, datetime
+
+# Specify the directory path
+csv_file_name = r'C:\Users\camil\Robotics\JoinedData\PNCMP2024-MatchExport-Q100.csv' #TODO:check if file name need changing later
+json_file_name = r'C:\Users\camil\Robotics\JoinedData\DCMPQualMatches.json'
+
+all_matches = []
+total_time = None
+current_time = datetime.datetime.hour*1000 + datetime.datetime.minute*100 + datetime.datetime.second + float(1/datetime.datetime.microsecond)
+
+with open(csv_file_name, newline='', encoding='utf-8') as csv_file:
+    csv_reader = csv.reader(csv_file)
+    first_row = True
+    for row in csv_reader:
+        if first_row:
+            column_names = []
+            for column_name in row:
+                column_names.append(column_name.strip('\uFEFF'))
+            first_row = False
+        else:
+            current_record = {}
+            for i in range(len(column_names)):
+                current_record[column_names[i]] = row[i]
+            all_matches.append(current_record)
+    total_time = datetime.datetime.hour*1000 + datetime.datetime.minute*100 + datetime.datetime.second + float(1/datetime.datetime.microsecond) - current_time
+
+
+#print(all_matches)
```

Comment:
Would be deleted, no commented code.

Reply:
Great catch, that's one of the issues
Also, I counted out all of the issues within the PR and theres about 13 of them total